data/_ui/tooltips.txt

```diff
@@ -360,6 +360,24 @@ tip "jump mass cost:"
 tip "jump base mass:"
 	`Modifies the impact your ship's mass has on the jump mass cost. Ships below this mass will have a cheaper jump cost than the base jump fuel cost, while ships above this mass will have more expensive jumps.`
 
+tip "jumpdrive damage:"
```

Comment:
Are there any other instance of a general "damage" attribute? Because this doesn't seem inline with the other damage attributes.

Reply:
I changed it to be more inline with for example firing hull

Reply:
I was referring to those damage attributes that do both shield and hull damage at the same time. They're not needed, you can just set the other two to the same value.

Reply:
No?
They dont do both at the same time, they get applied after each other, applying left over damage to hull, if shields are depleted

Reply:
oh yeah I didn't notice my bad sorry